Content/Items/Misc/Weapons.CoachGun.cs

```diff
@@ -62,6 +62,7 @@ public override void AddRecipes()
 			recipe.AddIngredient(ItemID.Musket, 1);
 			recipe.AddIngredient(ItemID.Explosives, 1);
 			recipe.AddTile(TileID.Anvils);
+			recipe.Register();
```

Comment:
good catch -- but this probably should have been a seperate ticket in the future. At the very least make sure to mention this in the PR message